syntaxes/test/dummy/expression.ng.tmLanguage-dummy.json

```diff
@@ -0,0 +1,4 @@
+{
+  "comment": "Dummy Angular expression TextMate grammar for use in testing",
+  "scopeName": "expression.ng"
+}
```

Comment:
I don't think we want this... we should test against the new grammar you provided :).